polyfill/intersection-observer.js

```diff
@@ -368,6 +368,10 @@ IntersectionObserver.prototype._computeTargetAndRootIntersection =
 
   while (!atRoot) {
     var parentRect = null;
+    var parentComputedStyle = window.getComputedStyle(parent);
```

Comment:
Yes, sorry - that wasn't a lot to go on.

I'm using it within WebComponents and the above change made everything stop working with an error on that line, I'll have to update again to get the error but the gist of it was that `getComputedStyle` was being called on something that didn't support it, I thought it was the document root.

I'll find the error and raise a proper issue ...